model/resource/cloud_provider/aws/ec2.yaml

```diff
@@ -0,0 +1,17 @@
+groups:
+  - id: aws.ec2
+    prefix: aws.ec2
+    type: resource
+    brief: >
+      Resources used by Amazon Elastic Compute Cloud (Amazon EC2).
+    attributes:
+      - id: instance.id
```

Comment:
@mx-psi Can you move the definition of this attribute to the registry and use a reference here? (see https://github.com/open-telemetry/semantic-conventions/pull/434/files for an example)